cfpb/consumerfinance.gov
# Add stylelint-config-prettier

## Additions

- Add `stylelint-config-prettier` to handle stylelint/prettier collisions.

## Changes

- Run Prettier against CSS/Less.
- Move `stylelint.config.cjs` to `.stylelintrc.cjs` to match eslint config naming.
- Spell out `eslint-config-prettier` in eslint config to make it clearer where that dependency is used.
- Remove prettier/stylelint colliding `function-parentheses-space-inside`, `max-empty-lines`, `indentation`, `media-feature-parentheses-space-inside`, and `selector-pseudo-class-parentheses-space-inside` rules.


## How to test this PR

1. PR checks should pass.

2 `yarn lint` should show no less file changes.

3. If curious, adding 

```js
"scripts": {
    "stylelint-check": "stylelint-config-prettier-check"
  }
```

To package.json and running `yarn stylelint-check` should show no collisions between prettier and stylelint.
